### DISCUSSION TITLE
TRUST4 on non-cancerous tissue samples ?

### DISCUSSION THREAD

**Discussion by micolak0115** [liulab-dfci/TRUST4#78] [Q&A] [answered]
Hi, thanks for the impressive tool you built.

Just a student playing around with whole blood samples using different tools.
I was wondering if it would be valid to use TRUST4 on the whole blood as well though.
The name of the pipeline and your discussion about tissue infiltration tell me that I should not intuitively. 
However, I do not see any problem according to the algorithm in the methods of your paper.
If not, could you please explain why it would be big no no to do so ???

Cheers,
Alan

**Comment by mourisl** [comment, marked as answer]
Yes, TRUST4 works on blood samples fine. There are T cells and B cells in the blood, so you will get TCRs and BCRs results too.

**Reply by micolak0115** [reply]
Thank you very much :)))
